### PR TITLE
Breadcrumb

### LINKED ISSUE #31: Add breadcrumb component
Also implement the same in website

### PR DESCRIPTION
**Pull request by Eyesnatcher-coder**
Hey @imabhinavagarwal , I have added the breadcrumb component in the website :).
Hope, you like!!

I am attaching the screenshot.
Screenshot 2022-10-04 190122

### CODE REVIEW

File: components/components.css [hunk 1]
```diff
@@ -354,3 +355,30 @@ iframe {
     color: var(--white);
 }
 
+ul.breadcrumb {
+    padding: 10px 16px;
+    list-style: none;
+    background-color: #0ea5e9;
```
Comment (imabhinavagarwal) [inline components/components.css]: define colors at one place and use in var(--color) format

File: components/components.html [hunk 2]
```diff
@@ -8,296 +8,312 @@
...
+                    style="border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </section>
+            <hr>
+            <section id="breadcrumb">
+                <h1>Breadcrumb</h1>
+                <p>Breadcrumbs are a secondary navigation aid that helps users easily understand the relation between
+                    their location on a page (like a product page) and higher level pages (a category page, for
+                    instance)</p>
+                <ul class="breadcrumb">
```
Comment (imabhinavagarwal) [inline components/components.html:312]: Implement the same at top of components page and keep it here as well. Goal is to keep it here as part of component but also use it in our website to beautify it

**Review by imabhinavagarwal — CHANGES_REQUESTED** [review]
Please resolve conflicts and fix review comments

File: components/components.html (hunk 2 above)
Comment (Eyesnatcher-coder) [reply components/components.html:312]: hey @imabhinavagarwal , Do you want website to look like
image
# Also tell if you want breadcrumb to be fixed  on top of website or not. 
# Do you want any javascript feature to be added to breadcrumb or not . Example should it get updated when you click to any component.
 :)

**Comment by Eyesnatcher-coder** [comment]
hey @imabhinavagarwal , Do you want website to look like 
image
* Also please tell that whetether you wanted breadcrumb component to be fixed in nature at top ?
* Do you want a feature to change the name on breadcrumb to change to the name of component when you click on any  component ?

**Comment by imabhinavagarwal** [comment]
> hey @imabhinavagarwal , Do you want website to look like this
 yes
> Also please tell that whetether you wanted breadcrumb component to be fixed in nature at top ?
No.
> Do you want a feature to change the name on breadcrumb to change to the name of component when you click on any  component ?
No

**Review by imabhinavagarwal — CHANGES_REQUESTED** [review]
I think the code is duplicated after you resolved conflicts.. please check and fix. You can always refer to preview generated

**Comment by Eyesnatcher-coder** [comment]
Hey @imabhinavagarwal , I have added the breadcrumb , please give review.

File: components/components.css [hunk 3]
```diff
@@ -598,3 +631,4 @@ input#img-6:checked ~ .carousel-dots label#img-dot-6 {
 
 
 input:checked + .slide-container .nav label { display: block; }
+>>>>>>> 3dc9622f247534f10574bad427f29d9aba03f4be
```
Comment (imabhinavagarwal) [inline components/components.css]: remove this

**Review by imabhinavagarwal — CHANGES_REQUESTED** [review]
address comments

File: components/components.css (hunk 3 above)
Comment (Eyesnatcher-coder) [reply components/components.css]: OK , I will do that.

**Review by Eyesnatcher-coder — COMMENTED** [review]
Review Done !

**Comment by Eyesnatcher-coder** [comment]
Hello @imabhinavagarwal , I have removed the address comment , please review and mark it as done.

**Review by imabhinavagarwal — APPROVED** [review]

**Pull request merged**